scripts/sixtyfps_weekly.py

```diff
@@ -0,0 +1,222 @@
+
+
+def parse_message(input: str) -> typing.Tuple[str, str]:
+    subject = ""
+    message = ""
+
+    for l in input.rstrip().split("\n"):
+        if not l.startswith("    "):
+            continue
+        if subject == "":
+            subject = l[4:]
+            continue
+
+        message += f"{l[4:]}\n"
+
+    return (subject, message.strip())
+
+
+def get_git_log(
+    start_date: datetime.datetime, end_date: datetime.datetime
+) -> typing.Sequence[typing.Tuple[str, str, str, str]]:
+    result = subprocess.run(
+        [
+            "/usr/bin/git",
+            "log",
+            f"--since={start_date.isoformat()}",
+            f"--until={end_date.isoformat()}",
+            "--pretty=format:%w(0)Subject: %s%nAuthor: %ae%nHash: %H%nBody:%n%w(80,4,4)%b%n%w(0)END_OF_BODY%n%n",
+        ],
+        capture_output=True,
+        check=True,
+    )
+
+    return _parse_git_log(result.stdout.decode("utf-8"))
+
+
+def _parse_git_log(log_data: str) -> typing.Sequence[typing.Tuple[str, str, str, str]]:
+    result: typing.List[typing.Tuple[str, str, str, str]] = []
+
+    author = ""
+    subject = ""
+    commit = ""
+    message = ""
+
+    for l in log_data.split("\n"):
+        if l == "":
+            message += "\n"
+        elif l.startswith("Subject: "):
+            subject = l[9:].strip()
+        elif l.startswith("Author: "):
+            author = l[8:].strip()
+        elif l.startswith("Hash: "):
+            commit = l[6:].strip()
+        elif l.startswith("Body:"):
+            message = ""
+        elif l.startswith("    "):
+            message = f"{message}{l[4:]}\n"
+        elif l.startswith("END_OF_BODY"):
+            message = message.strip()
+            message_str = message
+            message_str.replace("\n", "\\\n")
+
+            assert commit and author and subject
+
+            result.append((commit, author, subject, message))
+            commit = ""
+            author = ""
+            subject = ""
+            message = ""
+        else:
+            print(f'ERROR: Unknown line "{l}".')
+
+    return result
+
+
+def make_ordinal(n):
+    '''
+    Convert an integer into its ordinal representation::
+
+        make_ordinal(0)   => '0th'
+        make_ordinal(3)   => '3rd'
+        make_ordinal(122) => '122nd'
+        make_ordinal(213) => '213th'
+
+    This code was taken from
+    https://stackoverflow.com/questions/9647202/ordinal-numbers-replacement
+
+    under this license:
+    https://creativecommons.org/licenses/by-sa/4.0/
+    '''
+    n = int(n)
+    suffix = ['th', 'st', 'nd', 'rd', 'th'][min(n % 10, 4)]
+    if 11 <= (n % 100) <= 13:
+        suffix = 'th'
+    return str(n) + suffix
+
+
+def date_string(date: datetime.datetime) -> str:
+    month_names = [ "INVALID", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December" ]
+    return "{0} of {1} of {2}".format(make_ordinal(date.day), month_names[date.month], date.year)
```

Comment:
```suggestion
    return "{0} of {1} {2}".format(make_ordinal(date.day), month_names[date.month], date.year)
```

Also the start date don't include the year.  One should be consistent with what we already done